tigerpath/majors_and_certificates/majors/EAS.json

```diff
@@ -10,8 +10,8 @@
   "contacts": [
     {
       "type": "Chair",
-      "name": "Martin Kern",
-      "email": "mkern@princeton.edu"
+      "name": "Anna M. Shields",

```

Comment:
https://ua.princeton.edu/academic-units/department-east-asian-studies

Reply:
Could you also remove the Undergraduate Administrator entry below? Looks like that's outdated.

Reply:
Gotcha, fixed.